config/locales/en.yml

```diff
@@ -339,6 +339,8 @@ en:
       heading: Accessibility statement for Register trainee teachers
     data_requirements:
       heading: Check what data you need
+    search_schools:
```

Comment:
I don't think this is being used anywhere.

Reply:
yep, removed